config/settings.py

```diff
@@ -201,3 +213,33 @@
     MEDIA_ROOT = ""
 # END_FEATURE django_storages
 
+# START_FEATURE recommended_production_settings
+if not DEBUG:
+    import sentry_sdk
+    from sentry_sdk.integrations.django import DjangoIntegration
+    sentry_sdk.init(
+        dsn=env("SENTRY_DSN"),
+        integrations=[DjangoIntegration()],
+        traces_sample_rate=1.0,
+    )
+
+if not DEBUG:
```

Comment:
In general, we're trying to move to a model where we don't use debug to differentiate between local dev and server deployments.  I'd suggest creating a separate env setting `LOCAL` to switch SSL and the allowed hosts.  This is so that you can turn DEBUG off for local dev (sometimes necessary to test).